experiments/pyproject.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache 2.0"
 [tool.poetry.dependencies]
 python = ">=3.9,<3.13"
 black = "^23.11.0"
-numpy = "^1.26.1"
+numpy = "^1.26.1, <2.0"
```

Comment:
I don't know what came with numpy 2.0, so I won't use it until I fully understand potential breaking changes.